components/LocationComparisonSection/LocationComparisonBanner.tsx

```diff
@@ -22,8 +22,12 @@ const LocationComparisonBanner: FunctionComponent<ComparisonWrapperProps> = ({
   const [locations, setLocations] = useState<SpotlightLocation[]>([]);
 
   const onWidgetClick = (widgetState: boolean, location: SpotlightLocation | any): void => {
+    const index = locations.findIndex(x => x.name === location.name);
+    console.log('the index is ' + index);
     showActive(widgetState);
-    location.name ? setLocations([...locations, { name: location.name, geocode: location.geocode }]) : null;
+    location.name && index === -1
+      ? setLocations([...locations, { name: location.name, geocode: location.geocode }])
+      : null;
```

Comment:
Only use ternary operators when you're assigning a value, not as a condition for calling a function (unless that's part of an assignment operation). Use an if statement here.

Reply:
Replaced the ternary operator with an if statement